rentals.rb

```diff
@@ -0,0 +1,17 @@
+require_relative './person'
+require_relative './book'
+
+class Rental
+  attr_reader :book, :person
+  attr_accessor :date
```

Comment:
- Please ensure that the `:book, :person, :date` attributes all have the getter and setter methods, please fix this 👍 👍